source/leuven/tier2_hardware/mfa_login.rst

```diff
@@ -1,143 +1,244 @@
 
 - A valid private key
-- Access to a VSC associated university account
-
-The latter will be verified by prompting a VSC firewall link when trying to ssh to one of our KU Leuven clusters. When following this link, you will be asked to choose your university, and log in. If you already did that in a browser session (to access any other university-related websites), you do not need to log in again. If the login was succesfull, you will be redirected to a firewall.vscentrum.be page, with the message that your vsc identity was confirmed.
-
-The benefits of this method are:
-
-- keys without passphrase are less of a security problem
-- users can't share keys
-- users won't be able to connect if the university account is no longer valid
-
-What do you need to use MFA on your vsc-account
------------------------------------------------
-
-With our MFA enabled login nodes, you can access the cluster in following ways:
-
-- With ssh key + MFA prompt from IP addresses in any KU Leuven address range, any known LAN balancer address of other Flemish universities, any Belgian residential network, or a list of known institutes or organizations.
-- With ssh key only from IP addresses on which `VSC firewall page`_ is opened and that are also in a KU Leuven address range or a Belgian residential network.
-- With ssh key + confirmation from IP addresses on which `VSC firewall page`_ is opened and that are not in a KU Leuven address range and neither in a Belgian residential network. This confirmation is the same as the MFA prompt, but can also be answered by a popup on the HPC firewall page that the user has opened.
-- (only on login[-tier1].hpc.kuleuven.be) With ssh key + MFA prompt if connecting from any other IP address that was already seen during the last 90 days (so a user can open `VSC firewall page`_ manually once, and then connect each month directly with the MFA prompt).
-
-MFA Authentication without an agent
------------------------------------
-
-When you try to login to the cluster with a ssh-client (e.g. PuTTY, MobaXTerm...) and you have no agent running (see later):
-
-- start any SSH session to a KU Leuven cluster and authenticate with your private vsc-key
-- your ssh connection will prompt you a URL
-- copy/paste this URL in your browser and use the VSC account authentication
-
-Your SSH-connection will now be completed. This method also works from a Linux or Mac terminal.
-
-UI applications with SSH connection in the background
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Access to a VSC-associated university/institution account
+
+This document provides two methods to connect to the KU Leuven clusters with
+MFA. The first method, without an agent, is easier to get started but requires
+some repetitive steps with each connection. The second method, *with* an agent,
+requires some more effort to set up initially but will be easier when you need
+multiple connections or you are using some specific GUI applications to connect. 
```

Comment:
Would it be an idea to mention 'easier when you need... or **even necessary when** you are using some specific...'